src/components/Articles.jsx

```diff
@@ -0,0 +1,82 @@
+import {useEffect, useState} from 'react'
+import axios from 'axios';
+
+export default function Articles() {
+
+  const [articleList, setArticleList] = useState([])
+  const [pageNumber, setPageNumber] = useState(1)
+  const [totalArticles, setTotalArticles] = useState(0)
+  const [isFirstPage, setIsFirstPage] = useState(true)
+  const [isLastPage, setIsLastPage] = useState(false)
+
+  const pageLimit = Math.ceil(totalArticles / 10)
+
+  useEffect(() => {
+    axios.get(`https://news-api-so0z.onrender.com/api/articles?p=${pageNumber}`)
+      .then(({data}) => {
+        setArticleList(data.articles)
+        setTotalArticles(data.total_count)
+      })
+  }, [pageNumber])
+
+  function nextPage() {
+    if (pageNumber < pageLimit) {
+    setPageNumber(pageNumber + 1)
+    setIsFirstPage(false)
+    }
+    if (pageNumber === pageLimit - 1) {
+      setIsLastPage(true)
+    }
+  }  
+
+  function previousPage() {
+    if (pageNumber > 1) {
+    setPageNumber(pageNumber - 1);
+    } 
+    if (pageNumber === 2) {
+      setIsFirstPage(true)
+    }
+    if (pageNumber === pageLimit) {
+      setIsLastPage(false)
+    }
+  }
+  
+  return (
+    <>
+      <h2 id='top'>All Articles</h2>
+
+      <div className="flex-box">
+      <ul>
+        {articleList.map(article => {
+          const dateCreated = `${new Date(article.created_at)}`;
+
+          return (
+            <li key={article.article_id}>
+               <h3>{article.title}</h3>
+               <p>Topic: {article.topic}</p>
+               <p>By: {article.author}</p>
+               <p>Total Votes: {article.votes}</p>
+               <p>Total Comments: {article.comment_count}</p>
+               <p>Date created: {dateCreated}</p>
```

Comment:
Would be better to have the timestamp formatted in a bit more of a human readable way, than the default. Have a think how you could manipulate the data yourself to do this, or look into some packages that can do it for you 😄